src/OpenTelemetry/Metrics/HistogramBuckets.cs

```diff
@@ -32,6 +32,8 @@ public class HistogramBuckets
 
         internal double SnapshotSum;
 
+        internal int UsingHistogram = 0;
```

Comment:
`IsCriticalSectionOccupied` ? or something else? (not blocking, just noting if we can get better name)

Reply:
Renamed it to `IsCriticalSectionOccupied `.